File: javascript/1365_How_Many_Numbers_Are_Smaller_Than_the_Current_Number.js
var smallerNumbersThanCurrent = function(nums) {
    let resultArray = [];
    const sortedNums = [...nums].sort((a,b) => a-b);
    
    nums.forEach(num => {
        resultArray.push(sortedNums.indexOf(num))
    });
    return resultArray;
};

File: python/1365_How_Many_Numbers_Are_Smaller_Than_the_Current_Number.py
# https://leetcode.com/problems/how-many-numbers-are-smaller-than-the-current-number/
from typing import List

class Solution:
    def smallerNumbersThanCurrent(self, nums: List[int]) -> List[int]:
        sorted_nums = sorted(nums)
        return [sorted_nums.index(n) for n in nums]